src/ServiceInsight/Saga/SagaUpdateControl.xaml

```diff
@@ -377,35 +401,28 @@
                                    Foreground="{StaticResource SagaForeground}"
                                    Text="{Binding Name}"
                                    TextTrimming="CharacterEllipsis" />
-                        <TextBlock Foreground="{StaticResource SagaForeground}" Text=" =    " />
-                        <TextBlock FontWeight="Bold"
-                                   Foreground="{StaticResource SagaForeground}"
+                        <TextBlock Foreground="{StaticResource SagaSymbolsValue}" Text=" =    " />
+                        <TextBlock Foreground="{StaticResource SagaForeground}"
```

Comment:
@adamralph good point. I think that style doesn't provide any more value since we've removed other stuff. I think it's better to go back to just setting the foreground. I'll remove the style and make sure the usage is consistent.